week3/api-services/comprehensive_property_system.py

```
@@ -0,0 +1,765 @@
+        conn, db_time = get_db_connection()
+        cursor = conn.cursor()
+        cursor.execute("SELECT COUNT(*) FROM search_idx_parcel LIMIT 1")
+        property_count = cursor.fetchone()[0]
+        conn.close()
+        
+        health_status["components"]["database"] = {
+            "status": "healthy",
+            "response_time_ms": round(db_time, 2),
+            "property_count": property_count
+        }
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)